controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -46,50 +46,59 @@ controls:
     controls:
     - id: 5.2.1
       title: Minimize the admission of privileged containers
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_privileged_containers
       level: level_1
     - id: 5.2.2
       title: Minimize the admission of containers wishing to share the host process
         ID namespace
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_process_id_namespace
       level: level_1
     - id: 5.2.3
       title: Minimize the admission of containers wishing to share the host IPC namespace
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_ipc_namespace
       level: level_1
     - id: 5.2.4
       title: Minimize the admission of containers wishing to share the host network
         namespace
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_network_namespace
       level: level_1
     - id: 5.2.5
       title: Minimize the admission of containers with allowPrivilegeEscalation
-      status: pending
+      status: partial
       rules: []
+        - scc_limit_privilege_escalation
       level: level_1
     - id: 5.2.6
       title: Minimize the admission of root containers
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_root_containers
       level: level_2
     - id: 5.2.7
       title: Minimize the admission of containers with the NET_RAW capability
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_net_raw_capability
       level: level_1
     - id: 5.2.8
       title: Minimize the admission of containers with added capabilities
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_container_allowed_capabilities
       level: level_1
     - id: 5.2.9
       title: Minimize the admission of containers with capabilities assigned
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_drop_container_capabilities
       level: level_2
     - id: 5.2.10
       title: Minimize access to privileged Security Context Constraints
```

Comment:
We have a rule floating around that we used for STIG that we can reuse here I think?

CC: @jhrozek

Reply:
think you could use `rbac_least_privilege` here.

Reply:
wondering if we need to add a reference in the rule as well https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/rbac/rbac_least_privilege/rule.yml